refactor: moves config details out of quick start (#19)
brown-ccv/honeycomb-docs: docs/configuration.md

## Environment Variables

> Honeycomb uses environment variables during **build** and **run** time to control the different configurations. Below we list the  environment variables used by the app and indicate (via badges and text), whether they are mandatory, optional and needed during build (`npm build`) or run time (using the application executable or `npm run dev`). We also separet them via the enviroment (Desktop or Online) where get used by.

Reviewer: ```suggestion
Honeycomb uses environment variables during **build** and **run** time to control the different configurations. Below we list the  environment variables used by the app and indicate (via badges and text), whether they are mandatory, optional and needed during build (`npm build`) or run time (using the application executable or `npm run dev`). We also separate them based on the environment (Desktop or Online) where honeycomb will be run.
```

### Understanding Build Time and Run Time Variables

- ![runtime](https://img.shields.io/badge/-runtime-purple): Run-time environment variables allow for quick changes before running the task. They are convinient since they don't require building a whole different executable. For a Desktop Application, these are set at the system level and must be available at the time where the task runs. See [this tutorial for OS specific instructions.](https://www.imatest.com/docs/editing-system-environment-variables/#Windows). For a Firebase Application, these must be stored in the Firestore.

- ![buildtime](https://img.shields.io/badge/-buildtime-blue): Build-time environment variable are configured before the application is built and cannot be changed at run-time. They are configured via `.env` files before building. Instead of using a single `.env` file, we prefer to defined separate files under the `env/` directory, and use a package called `dotenv-cli` before building`. 

> The `dotenv-cli` commes with the `dotenv` command that can be used to propery load the needed variables. This command does not run directly from the command line (terminal). Instead, to set build-time environment variables, we need to add a  script to `package.json` with the following format:

Reviewer 2: is this supposed to be `comes` instead of `commes`

Author: Corrected, along with few other typos!
